site/specs-source/voice.json

```diff
@@ -3012,171 +3077,207 @@
           }
         }
       },
       "ConferenceRecordingMetadata": {
         "type": "object",
         "properties": {
           "accountId": {
-            "type": "string"
+            "type": "string",
+            "description": "Your account ID"
           },
           "conferenceId": {
-            "type": "string"
+            "type": "string",
+            "description": "The unique, Bandwidth-generated ID of the conference that was recorded"
           },
           "name": {
-            "type": "string"
+            "type": "string",
+            "description": "The user-specified name of the conference that was recorded"
           },
           "recordingId": {
-            "type": "string"
+            "type": "string",
+            "description": "The unique ID of this recording"
           },
           "duration": {
             "type": "string",
-            "description": "Format is ISO-8601"
+            "description": "The duration of the recording in ISO-8601 format"
           },
           "channels": {
             "type": "integer",
-            "format": "int32"
+            "format": "int32",
+            "description": "Always `1` for conference recordings; multi-channel recordings are not supported on conferences."
           },
           "startTime": {
             "type": "string",
-            "format": "date-time"
+            "format": "date-time",
+            "description": "The time that the recording started in ISO-8601 format"
           },
           "endTime": {
             "type": "string",
-            "format": "date-time"
+            "format": "date-time",
+            "description": "The time that the recording ended in ISO-8601 format"
           },
           "fileFormat": {
             "type": "string",
             "enum": [
               "mp3",
               "wav"
-            ]
+            ],
+            "description": "The format that the recording is stored in"
           },
           "status": {
             "type": "string",
             "description": "The current status of the recording. Current possible values are 'processing', 'partial', 'complete', 'deleted', and 'error'. Additional states may be added in the future, so your application must be tolerant of unknown values."
           },
           "mediaUrl": {
+            "nullable": true,
             "type": "string",
-            "format": "uri"
+            "format": "uri",
+            "description": "The URL that can be used to download the recording. Only present if the recording is finished and may be downloaded."
           }
         }
       },
       "MachineDetectionConfiguration": {
```

Comment:
Can you put the default value in `detectionTimeout`? It's 15s.

Reply:
Done